### ISSUE TITLE
Sinichi et al --- Quality in Question: Assessing the accuracy of four heart rate wearables and the implications for psychophysiological research

### ISSUE THREAD

**Issue by sje30** [codecheckers/register#134]
**Repository**: https://github.com/yiquintero/wearable-hrv

**Certificate**: `2024-019` 

Amin Sinichi - [m.sinichi@vu.nl](mailto:m.sinichi@vu.nl) (ORCID: 0009-0008-2491-1542)
Pre-print: https://doi.org/10.31219/osf.io/wkzsn
Data and materials: https://osf.io/y2q3r/
Code (a Python package I developed through these analyses): https://github.com/Aminsinichi/wearable-hrv

Gabriella Low Chew Tung - [G.M.LowChewTung@student.tudelft.nl](mailto:G.M.LowChewTung@student.tudelft.nl)
Roel Janssen - [r.r.e.janssen@tudelft.nl](mailto:r.r.e.janssen@tudelft.nl) (0000-0003-4324-5350)
Yasel Quintero - [y.i.quinterolares@tudelft.nl](mailto:y.i.quinterolares@tudelft.nl)
Joslyn Sun - [J.S.Sun@tudelft.nl](mailto:J.S.Sun@tudelft.nl) (https://orcid.org/0009-0002-2379-0060)
Tornike Skhirtladze [skhirtladze.tornike@gmail.com](mailto:skhirtladze.tornike@gmail.com)

**Comment by nuest** [comment]
@sje30 Is this a placeholder for https://github.com/codecheckers/DementiaPhysicalActivity ?

**Comment by sje30** [comment]
> @sje30 Is this a placeholder for https://github.com/codecheckers/DementiaPhysicalActivity ?

No... coincidentally we had two related papers on wearables!  I'm working on the meta data now...

**Comment by sje30** [comment]
published:

**Comment by sje30** [comment]
The paper relating to this codecheck has now been published

http://dx.doi.org/10.1111/psyp.70004

**Comment by nuest** [comment]
Yes, I would say that we prepare a new PDF on Zenodo and then we update the codecheck.yml on our end with the version-specific Zenodo DOI.

Can I support that in any way?
